content/en/tools/starter-kits.md

```diff
@@ -23,6 +23,7 @@ Know of a Hugo-related starter kit that isn't mentioned here? [Please add it to
 The following starter kits are developed by active members of the Hugo community. If you find yourself having issues with any of the projects, it's best to file an issue directly with the project's maintainer(s).
 {{% /note %}}
 
+* [Hugo Wrapper][hugow]. Hugow Wrapper is a POSIX-style shell script act as a wrapper of Hugo binary which is, in fact, [platform dependent][hugow-platform]. It can be executed in variety of [Operating Systems][hugow-test] and [Command Shells][hugow-test].
```

Comment:
How about:

"Hugow Wrapper is a POSIX-style shell script which acts as a wrapper to download and run the Hugo binary for your platform. It can be executed in variety of [Operating Systems][hugow-test] and [Command Shells][hugow-test]."

So you do not need the "hugow-platform" link then (also that link alias is incorrect.. it has nothing to do with hugow).